excise/tests/cassettes/test_preprocess_order_creation.yaml

```diff
@@ -1,10 +1,10 @@
 interactions:
 - request:
-    body: '{"EffectiveDate": "02/18/21", "InvoiceDate": "02/18/21", "TitleTransferCode":
-      "DEST", "TransactionType": "RETAIL", "TransactionLines": [{"InvoiceLine": 9,
-      "ProductCode": "123", "UnitPrice": "10.000", "UnitOfMeasure": "PAC", "BilledUnits":
+    body: '{"EffectiveDate": "2021-05-10", "InvoiceDate": "2021-05-10", "TitleTransferCode":
```

Comment:
This cassette has 2 requests. Are we sure that we should send 2 requests to ATE in this function?